package-system/AWSGameLiftServerSDK/FindAWSGameLiftServerSDK.cmake.Linux

```diff
@@ -14,30 +14,21 @@ if (TARGET ${TARGET_WITH_NAMESPACE})
     return()
 endif()
 
-if ("${OPENSSL_VERSION}" STREQUAL "")
-    message(FATAL_ERROR "OpenSSL not detected. The OpenSSL dev package is required for O3DE")
-elseif ("${OPENSSL_VERSION}" VERSION_LESS "3.0.0")
-    set(SRC_OPENSSL_BASE openssl-1)
-else()
-    set(SRC_OPENSSL_BASE openssl-3)
-endif()
-
-
 set(LIB_NAME "AWSGameLiftServerSDK")
 
 set(${LIB_NAME}_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/${LIB_NAME}/include)
 
 if (LY_MONOLITHIC_GAME)
-    set(AWSGAMELIFTSERVERSDK_LIB_PATH ${CMAKE_CURRENT_LIST_DIR}/${LIB_NAME}/lib/${SRC_OPENSSL_BASE})
+    set(AWSGAMELIFTSERVERSDK_LIB_PATH ${CMAKE_CURRENT_LIST_DIR}/${LIB_NAME}/lib)
 
     set(AWSGAMELIFTSERVERSDK_LIBS
-        ${AWSGAMELIFTSERVERSDK_LIB_PATH}/libaws-cpp-sdk-gamelift-server.a
+        ${AWSGAMELIFTSERVERSDK_LIB_PATH}/libaws-cpp-sdk-gamelift-server${CMAKE_STATIC_LIBRARY_SUFFIX}
```

Comment:
nitpick: Should we also replace "lib" with [${CMAKE_STATIC_LIBRARY_PREFIX}](https://cmake.org/cmake/help/latest/variable/CMAKE_STATIC_LIBRARY_PREFIX.html)
```suggestion
        ${AWSGAMELIFTSERVERSDK_LIB_PATH}/${CMAKE_STATIC_LIBRARY_PREFIX}aws-cpp-sdk-gamelift-server${CMAKE_STATIC_LIBRARY_SUFFIX}
```